react-frontend/src/components/StyleComponents/nav.js

```diff
@@ -21,3 +93,22 @@ export const NavTitle = styled.p.attrs({
     margin: 0px 5px 0px -5%;
   }
 `;
+
+export const SkipToMainContent = styled.a.attrs({
+  className: 'skipToMainContent',
+  accesskey: '0',
+  href: '#main-content-anchor',
+  'aria-label': 'Skip to main content',
+})`
+  position: absolute;
+  left: -10000px;
+  width: 120px;
+  color: #fcba19;
+  :hover {
+    color: #fcba19;
+  }
+  :focus {
+    position: static;
+    height: auto;
+  }
+`;
```

Comment:
I have submitted some styling improvements for the "Skip to main content" link in #351.

Reply:
I have incorporated them.